CMakeLists.txt

```
@@ -90,43 +109,31 @@ configure_file(src/version.F90.in version.F90 @ONLY)
 
 add_executable(tuv-x src/tuvx.F90 version.F90)
 
-set_target_properties(tuv-x
-  PROPERTIES
-  LINKER_LANGUAGE Fortran
-)
+set_target_properties(tuv-x PROPERTIES LINKER_LANGUAGE Fortran)
 
 if(NOT BUILD_SHARED_LIBS)
-  set_target_properties(tuv-x
-    PROPERTIES
-    POSITION_INDEPENDENT_CODE ON
-  )
+  set_target_properties(tuv-x PROPERTIES POSITION_INDEPENDENT_CODE ON)
 endif()
 
-target_link_libraries(tuv-x 
-  PUBLIC 
-    musica::tuvx
-    yaml-cpp::yaml-cpp
-    ${BLAS_LIBRARIES}
-    ${LAPACK_LIBRARIES}
-)
+target_link_libraries(tuv-x PUBLIC musica::tuvx yaml-cpp::yaml-cpp
+                                   ${BLAS_LIBRARIES} ${LAPACK_LIBRARIES})
```

```suggestion
target_link_libraries(tuv-x PUBLIC musica::tuvx yaml-cpp::yaml-cpp)
```

If you would, please publicly link `${BLAS_LIBRARIES} ${LAPACK_LIBRARIES}` to musica::tuvx. We wouldn't need to add links to `${BLAS_LIBRARIES} ${LAPACK_LIBRARIES}` anywhere else in the cmake files if we do that